packages/node/src/handlers/call-api.ts

```diff
@@ -36,8 +26,12 @@ function addMetadataParameters(
         _airnode_chain_type: chain.type,
         _airnode_airnode_rrp: chain.contracts.AirnodeRrp,
       };
+      return {
```

Comment:
No need to spread here

```suggestion
      return metadataParametersV1;
```

Reply:
I think I do because I said that the `addMetadataParameters` method returns `adapter.Parameters` object and `metadataParametersV1` is of type `adapter.RelayMetadataV1`.

Maybe I overcomplicated things with this type checking but I was trying to make it more future proof by adding `adapter.RelayMetadataV1`. In the future we might have `adapter.RelayMetadataV2` interface etc and I wanted to have some sort of type checking to maker sure the properties we are adding are the correct ones.

Reply:
I could maybe make `adapter.RelayMetadataV1` extend `adapter.Parameters` and the compiler would tell if a property is missing but it won't complain if we are adding more props to the object.

Reply:
I'll merge to unblock an API integration but I could still improve this afterwards.